packages/router/src/link.tsx

```diff
@@ -5,17 +5,18 @@
 
 import React, { forwardRef } from 'react'
 
-import { navigate } from './history'
+import { NavigateOptions, navigate } from './history'
```

Comment:
```suggestion
import { navigate } from './history'
import type { NavigateOptions } from './history'`
```

And same thing on the other places this error shows up